packages/x-date-pickers/src/PickersActionBar/PickersActionBar.test.tsx

```diff
@@ -85,17 +82,17 @@ describe('<PickersActionBar />', () => {
       />,
     );
 
-    fireUserEvent.mousePress(screen.getByText(/ok/i));
+    await user.click(screen.getByText(/ok/i));
     expect(onAccept.callCount).to.equal(1);
   });
 
-  it('should render button for "today" action calling the associated callback', () => {
+  it('should render button for "today" action calling the associated callback', async () => {
     const onAccept = spy();
     const onClear = spy();
     const onCancel = spy();
     const onSetToday = spy();
 
-    render(
+    const { user } = render(
```

Comment:
This looks off the direction set in: https://github.com/mui/mui-public/issues/173

```suggestion
    const render(
```

Reply:
The `user` from `render` comes from `userEvent.setup()`, we should probably validate if there's something important done in that setup call:

https://github.com/mui/material-ui/blob/eea5079a00bedb68e27f22b7f94d9a71cc97ee1e/packages-internal/test-utils/src/createRenderer.tsx#L304

Reply:
@oliviertassinari  `@testing-library/react` doesn't have a `user` export. They [advise](https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent) to initialize a new instance for every test which is what we do in our `render` method wrapper.

Personally, I find overly abstracting things in tests leads to harder to maintain code. Wrappers tend to want to grow indefinitely as the project progresses. But since in core the wrapper was there and used everywhere, it guess it made sense to fold the `userEvent` code into it. For tests I believe the DRY principle sometimes is harmful. People end up writing a bunch of abstractions over their testing code and they end up becoming just as hard to refactor as the main code.

Reply:
> we should probably validate if there's something important done in that setup call:

If necessary, you can chain multiple setup calls with different options, see https://github.com/mui/mui-x/pull/14142/files#diff-c307c02f110006557e5c75d52b6f6eef030b543ba4b193c936cce9909902f930R113-R115

Reply:
We should open a new issue to discuss improving the current `user` setup. It seems out of topic here.

Reply:
>  we should probably validate if there's something important done in that setup call:

Setup doesn't seem to do a lot https://github.com/testing-library/user-event/blob/d0362796a33c2d39713998f82ae309020c37b385/src/setup/setup.ts#L82. I was worried about the performance slowness of running in each test, but it doesn't seem that bad. Not speeding the CI though 😄. 

> They [advise](https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent) to initialize a new instance for every test which is what we do in our render method wrapper.

In the why they advice against it, they seems to point against using beforeEach, but it's not what we would do here?

> We should open a new issue to discuss improving the current user setup. It is out of topic here.

I think we can use https://github.com/mui/mui-public/issues/173 for this.

Reply:
> We recommend invoking userEvent.setup() before the component is rendered. This can be done in the test itself, or by using a setup function. We discourage rendering or using any userEvent functions outside of the test itself - e.g. in a before/after hook - for reasons described in "Avoid Nesting When You're Testing".

afaiu, what we do falls under "by using a setup function".